### PR TITLE
Update servicebus to enable live testing in sovereign clouds for multiple services

### PR DESCRIPTION
**Pull request by Luyunmt**
These changes contain fixes for the servicebus to be able to run live tests against AzureUsGovernment and AzureChina clouds. The sovereign cloud live tests will be green with the changes.
@benbp @danieljurek @jongio

### CODE REVIEW

File: sdk/servicebus/tests.data.yml [hunk 1]
```diff
@@ -8,3 +8,4 @@ extends:
     SDKType: data
     TimeoutInMinutes: 120
```
Comment (jsquire) [inline sdk/servicebus/tests.data.yml:9]: I'd recommend bumping these up to 240 in both files;  the Service Bus suite uses dynamic resources to isolate tests and the management plane responsiveness in the China cloud is very, very slow.

**Review by jsquire — APPROVED** [review]

File: sdk/servicebus/tests.data.yml [hunk 2]
```diff
@@ -6,5 +6,6 @@ extends:
     MaxParallel: 4
     ServiceDirectory: servicebus
     SDKType: data
-    TimeoutInMinutes: 120
+    TimeoutInMinutes: 240
     Clouds: 'Public,Canary'
+    SupportedClouds: 'Public,Preview,UsGov,China,Canary'
```
Comment (benbp) [inline sdk/servicebus/tests.data.yml:11]: Let's remove `Preview` from this, as I don't think we need it.

**Review by benbp — CHANGES_REQUESTED** [review]

**Review by benbp — APPROVED** [review]

**Comment by benbp** [comment]
Looks good to me!

**Pull request merged**